lib/msf/core/payload/pingback.rb

```diff
@@ -0,0 +1,51 @@
+# -*- coding => binary -*-
+
+require 'msf/core'
+require 'msf/core/module/platform'
+require 'rex/text'
+
+#
+# This class provides methods for calculating, extracting, and parsing
+# unique ID values used by payloads.
+#
+module Msf::Payload::Pingback
+
+  #
+  # Instance methods
+  #
+    #
+  # Generates a URI with a given checksum and optionally with an embedded UUID if
```

Comment:
What is this comment block for? The `generate_pingback_uuid` doesn't appear to take any parameters.